README.md

```diff
@@ -12,12 +12,12 @@ record the decision, and collect statistics.
 - Rspec features, not cucumber
 - Slim templating language
 - Foundation view framework
-- Coffeescript in preference to Javascript
+- JavaScript in preference to Coffeescript
```

Comment:
I'm aware it is mainly personal preference, this. But, it made sense to pick one or the other.